src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/index.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends '@SyliusAdmin/Shared/Layout/base.html.twig' %}
+
+{% set definition = resources.definition %}
+{% set data = resources.data %}
+{% set event_name = metadata.applicationName ~ '.admin.' ~ 'index.body' %}
+{% set resource_event_name = metadata.applicationName ~ '.admin.' ~ metadata.name ~ '.index.body' %}
+
+{% set header = configuration.vars.header|default(metadata.applicationName~'.ui.'~metadata.pluralName) %}
+
+{% block title %}{{ header|trans }} {{ parent() }}{% endblock %}
+
+{% block body %}
+    {{ sylius_template_event(event_name, { event_prefix: event_name, resource_event_prefix: resource_event_name }) }}
```

Comment:
You have an issue here, cause you remove the fallback for 'sylius.admin.index.body' when you have'nt define 'app.admin.index.body' (for custom entities).